### ISSUE TITLE
AMRFinderPlus database cannot be found

### ISSUE THREAD

**Issue by lsayaved** [oschwengers/bakta#151]
Hi,

I installed Bakta 1.5.1 through conda and docker, and both are showing the same error:

ERROR: AMRFinderPlus database not installed! Please, install AMRFinderPlus's internal database by executing: 'amrfinder_update --database <database-path>/amrfinderplus-db'. This must be done only once.

I've downloaded the amr database following the above command without success and I also tried downloading again the database using the command bakta_db download --output <>. Am I missing something?

Thanks!
Lizbeth

**Comment by oschwengers** [comment]
Hi @lsayaved,
thanks for reaching out. I'd need a little bit more information on which exact commands did you execute and what are the exact error messages or what do you exactly mean by "without success"?

Please, be cautious on the db paths as in your example  above, you tell AMRFinderPlus to write the database to `/amrfinderplus-db` which is located in the root directory which will not be allowed on a multi-user system and would certainly not be considered as best practices anyways.

You can either download the Bakta database along with the AMRFinderPlus database by `bakta_db download --output <DB_PATH>` or you download the database yourself, deflate the `.tar.gz` file and manually download the AMRFinderPlus db by yourself via `amrfinder_update --database <DB_PATH>/amrfinderplus-db`.

Best regards

**Comment by lsayaved** [comment]
Hi, Thanks for your reply:

The exact commands were:
>conda install -c conda-forge -c bioconda bakta=1.5.1

>source activate bakta

>bakta --db ~/remote/slurm_narbad/db/bakta/db/db/  --threads 8 QI0013_nocat.fas
ERROR: AMRFinderPlus database not installed! Please, install AMRFinderPlus's internal database by executing: 'amrfinder_update --database <database-path>/amrfinderplus-db'. This must be done only once.

>amrfinder_update --database ~/remote/slurm_narbad/db/bakta/db//db/amrfinderplus-db --force_update

>bakta --db ~/remote/slurm_narbad/db/bakta/db/db/  --threads 8 QI0013_nocat.fas
ERROR: AMRFinderPlus database not installed! Please, install AMRFinderPlus's internal database by executing: 'amrfinder_update --database <database-path>/amrfinderplus-db'. This must be done only once.

_I tried also downloading the database:_
>bakta_db download --output ~/remote/slurm_narbad/shared_tools/bakta_db
Bakta software version: 1.5.0
Required database schema version: 4

fetch DB versions...
        ... compatible DB versions: 1
download database: v4.0, 2022-08-29, DOI: 10.5281/zenodo.7025248, URL: https://zenodo.org/record/7025248/files/db.tar.gz...
|████████████████████████████████████████| 31734559/31734559 [100%] in 10:34.9 (49987.42/s)
        ... done
check MD5 sum...
        ...database file OK: 4116192902e88140b19b5e4d88be9fa1
extract DB tarball: file=/home/ubuntu/remote/slurm_narbad/shared_tools/bakta_db/db.tar.gz, output=/home/ubuntu/remote/slurm_narbad/shared_tools/bakta_db
successfully downloaded Bakta database!
        version: 4.0
        DOI: 10.5281/zenodo.7025248
        path: /home/ubuntu/remote/slurm_narbad/shared_tools/bakta_db/db
update AMRFinderPlus database...
        ... done

Run Bakta using '--db /home/ubuntu/remote/slurm_narbad/shared_tools/bakta_db/db' or set a BAKTA_DB environment variable: 'export BAKTA_DB=/home/ubuntu/remote/slurm_narbad/shared_tools/bakta_db/db'

>bakta --db /home/ubuntu/remote/slurm_narbad/shared_tools/bakta_db/db  --threads 8 QI0013_nocat.fas
ERROR: AMRFinderPlus database not installed! Please, install AMRFinderPlus's internal database by executing: 'amrfinder_update --database <database-path>/amrfinderplus-db'. This must be done only once.

**Comment by oschwengers** [comment]
Hmm, interesting! Could you please provide the content of the database directories? Actually, they should exist but for some reason Bakta is either unable to detect them or something is missing or has maybe changed.

```
ls -l /home/ubuntu/remote/slurm_narbad/shared_tools/bakta_db/db/amrfinderplus-db/
```
and 
```
ls -l /home/ubuntu/remote/slurm_narbad/shared_tools/bakta_db/db/amrfinderplus-db/latest/
```

**Comment by lsayaved** [comment]
Oh, that must be the issue. I only had a folder called 2022-08-09.1 by default. I've renamed it to latest and now it works.

**Comment by oschwengers** [comment]
OK, I see. Still a little bit confusing, since AMRFinder should create both, the actual database directory named by the timestamp and a symbolic link *latest* linked to this directory.

**Comment by cizydorczyk** [comment]
I just had this same issue on slurm but turned out it was the symbolic link that was causing problems. I re-created the link and it works just fine now.

I had initially uploaded the db, rather than downloading directly to my server. I suppose the link broke during the upload, and I had to recreate it manually. AMRFinder was not replacing the broken link, but I did not delete it and try again; may that's expected behavior.

Conrad
